File: 241008/공약수의 유무/presence-or-absence-of-a-common-divisor.py
inp = input().split()
a = int(inp[0])
b = int(inp[1])

state = 0

for i in range(a, b+1):
    if (1920 % i == 0 and 2880 % i == 0):
        state = 1

if (state == 1):
    print(1)
else:
    print(0)

File: 241008/공약수의 유무/presence-or-absence-of-a-common-divisor.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split(" ").map(Number);

const a = input[0];
const b = input[1];

let state = 0;

for (let i = a; i <= b; i++) {
    if (1920 % i === 0 && 2880 % i === 0) {
        state = 1;
    }
}

if (state === 1) {
    console.log(1);
} else {
    console.log(0);
}